prog/setup_hugepages.rb

```diff
@@ -5,8 +5,8 @@ class Prog::SetupHugepages < Prog::Base
 
   label def start
     hugepage_size = "1G"
-    # put away 1 core of overhead for host, and reserve 1G for SPDK
-    hugepage_cnt = 1 + (vm_host.total_mem_gib * (vm_host.total_cores - 1)) / vm_host.total_cores
+    # put away 1 core of overhead for host, and reserve 2x1G for 2 SPDK installations
```

Comment:
Note, we already can't get all the hugepages we ask for on the RX220 arm64, we may be pushing our luck here. But it's probably going to good enough for now.